src/test/java/com/google/api/codegen/testdata/csharp_gapic_client_library.baseline

```diff
@@ -3491,14 +3507,27 @@ namespace Google.Example.Library.V1
         /// <returns>
         /// A Task containing the RPC response.
         /// </returns>
-        public virtual Task<Operation> GetBigBookAsync(
+        public virtual Task<Operation<Book>> GetBigBookAsync(
             GetBookRequest request,
             CallSettings callSettings = null)
         {
             throw new NotImplementedException();
         }
 
         /// <summary>
+        /// Asynchronously poll an operation once, using an <c>operationName</c> from a previous invocation of <see cref="GetBigBookAsync"/>.
+        /// </summary>
+        /// <param name="operationName">The name of a previously invoked operation. Must not be <c>null</c> or empty.</param>
+        /// <param name="callSettings">If not null, applies overrides to this RPC call.</param>
+        /// <returns>A task representing the result of polling the operation.</returns>
+        public virtual Task<Operation<Book>> PollOnceGetBigBookAsync(
```

Comment:
I thought we were going to name this based on the return type rather than the operation type? Definitely something to chat about - hopefully won't be too hard to code when we've worked out what we want.